Move creating a new worker type tutorial to guides (#9455)
PrefectHQ/prefect: docs/concepts/work-pools.md

| [`kubernetes`](https://prefecthq.github.io/prefect-kubernetes/worker/) | Executes flow runs as Kubernetes jobs. | `prefect-kubernetes` |
| [`docker`](https://prefecthq.github.io/prefect-docker/worker/) | Executes flow runs within Docker containers. | `prefect-docker` |

> If you don’t see a worker type that meets your needs, you can follow [the guide](/guides/deployment/developing-a-new-worker-type/) to create one.

Reviewer: ```suggestion
If you don’t see a worker type that meets your needs, consider [developing a new worker type](/guides/deployment/developing-a-new-worker-type/)!
```

@billpalombi might have better verbiage for this

Reviewer 2: I like that!
